### ISSUE TITLE
The system requirements are not satisfied.

### ISSUE THREAD

**Issue by KobeClearlove** [intel/haxm#407]
Version: 7.7.0
Output folder: C:\Program Files\Intel\HAXM
Extract: checktool.exe... 100%
Execute: C:\Program Files\Intel\HAXM\checktool.exe --verbose
The system requirements are not satisfied.

**Comment by KobeClearlove** [comment]
![image](https://user-images.githubusercontent.com/83992297/133425672-0a5975fd-b061-476b-b9a8-2fcb60c973e1.png)

**Comment by wcwang** [comment]
Thanks for your report. Could you find the installation log in %TEMP% folder and check your system configuration according to the [wiki page](https://github.com/intel/haxm/wiki/Windows-Installation-Environment-Check-and-Configurations)?

**Comment by KobeClearlove** [comment]
![image](https://user-images.githubusercontent.com/83992297/133428280-5d646ee0-5bd8-4421-969b-e9ba306aed7b.png)
Essentials
Product Collection
9th Generation Intel® Core™ i7 Processors

Code Name
Products formerly Coffee Lake

Vertical Segment
Mobile

Processor Number
i7-9750H

Status
Launched

Launch Date
Q2'19

Lithography
14 nm

Recommended Customer Price
$395.00
CPU Specifications
# of Cores
6

# of Threads
12

Processor Base Frequency
2.60 GHz

Max Turbo Frequency
4.50 GHz

Cache
12 MB Intel® Smart Cache

Bus Speed
8 GT/s

TDP
45 W

Configurable TDP-down
35 W

Supplemental Information
Embedded Options Available
No

Datasheet
View now

Memory Specifications
Max Memory Size (dependent on memory type)
128 GB

Memory Types
DDR4-2666, LPDDR3-2133

Max # of Memory Channels
2

Max Memory Bandwidth
41.8 GB/s

ECC Memory Supported ‡
No

Processor Graphics
Processor Graphics ‡
Intel® UHD Graphics 630

Graphics Base Frequency
350 MHz

Graphics Max Dynamic Frequency
1.15 GHz

Graphics Video Max Memory
64 GB

Graphics Output
eDP/DP/HDMI/DVI

4K Support
Yes, at 60Hz

Max Resolution (HDMI)‡
4096 x 2304@30Hz

Max Resolution (DP)‡
4096 x 2304@60Hz

Max Resolution (eDP - Integrated Flat Panel)‡
4096 x 2304@60Hz

Max Resolution (VGA)‡
N/A

DirectX* Support
12

OpenGL* Support
4.5

Intel® Quick Sync Video
Yes

Intel® InTru™ 3D Technology
Yes

Intel® Clear Video HD Technology
Yes

Intel® Clear Video Technology
Yes

# of Displays Supported ‡
3

Device ID
0x3E9B

Expansion Options
PCI Express Revision
3.0

PCI Express Configurations ‡
Up to 1x16, 2x8, 1x8+2x4

Max # of PCI Express Lanes
16

Package Specifications
Sockets Supported
FCBGA1440

Max CPU Configuration
1

TJUNCTION
100°C

Package Size
42mm x 28mm

Advanced Technologies
Intel® Optane™ Memory Supported ‡
Yes

Intel® Speed Shift Technology
Yes

Intel® Thermal Velocity Boost
No

Intel® Turbo Boost Technology ‡
2.0

Intel vPro® Platform Eligibility ‡
No

Intel® Hyper-Threading Technology ‡
Yes

Intel® Virtualization Technology (VT-x) ‡
Yes

Intel® Virtualization Technology for Directed I/O (VT-d) ‡
Yes

Intel® VT-x with Extended Page Tables (EPT) ‡
Yes

Intel® Transactional Synchronization Extensions
No

Intel® 64 ‡
Yes

Instruction Set
64-bit

Instruction Set Extensions
Intel® SSE4.1, Intel® SSE4.2, Intel® AVX2

Intel® My WiFi Technology
Yes

Idle States
Yes

Enhanced Intel SpeedStep® Technology
Yes

Thermal Monitoring Technologies
Yes

Intel® Flex Memory Access
Yes

Intel® Identity Protection Technology ‡
Yes

Intel® Stable Image Platform Program (SIPP)
No

Security & Reliability
Intel® AES New Instructions
Yes

Secure Key
Yes

Intel® Software Guard Extensions (Intel® SGX)
Yes with Intel® ME

Intel® Memory Protection Extensions (Intel® MPX)
Yes

Intel® OS Guard
Yes

Intel® Trusted Execution Technology ‡
No

Execute Disable Bit ‡
Yes

**Comment by KobeClearlove** [comment]
It is seen supported by all

**Comment by wcwang** [comment]
Could you run below command in Command Prompt and check if the error level is 0?
> "C:\Program Files\Intel\HAXM\checktool.exe" --verbose
> echo %ERRORLEVEL%

Then try to execute below command in an elevated Command Prompt to disable Hyper-V and reboot your Windows.

> bcdedit /set hypervisorlaunchtype off

**Comment by KobeClearlove** [comment]
![image](https://user-images.githubusercontent.com/83992297/133438074-cf91d7b0-ae20-4a5b-a2ef-7eca44ae87d2.png)

**Comment by wcwang** [comment]
You are right. All configurations are met to install HAXM. Could you run below command under the folder of the extracted release package in the elevated Command Prompt?

> silent_install.bat

**Comment by KobeClearlove** [comment]
![image](https://user-images.githubusercontent.com/83992297/133440345-a7f55a15-4ca9-42e2-8bb8-85b6aebd10b8.png)

**Comment by wcwang** [comment]
Thanks for your reply. Sorry for bringing you inconvenience. We will investigate this weird issue soon.

**Comment by KobeClearlove** [comment]
So I can't install haxm normally?

**Comment by wcwang** [comment]
Have you ever enabled WSL2? Could you run below command in an elevated PowerShell and reboot again?

`> Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux`

**Comment by KobeClearlove** [comment]
![image](https://user-images.githubusercontent.com/83992297/133450152-7260fb6a-62a8-4dbe-80d1-d85529288919.png)

**Comment by srepollock** [comment]
I am also unable to install haxm on windows; getting the same error that "Hyper-V is not disabled" but there is no way to enable/disable through settings. I do have WSL turned on and I cannot turn it off.

**Comment by macena1989** [comment]
Is your virtualization enabled?  Run the command `systeminfo ` in the prompt and check Hyper-V Requirements.
I was having the same issue, after i enabled virtualization the installation worked. Have a look here: 
https://mashtips.com/enable-virtualization-windows-10/
You can enable it in the BIOS setup.

**Comment by wayne-ma** [comment]
@macena1989 ,please try 'bcdedit /set hypervisorlaunchtype off' in prompt window with admin and reboot you PC, then try again.

**Comment by srepollock** [comment]
Hyper-V is not an option for Windows 10 Home

**Comment by srepollock** [comment]
> @srepollock ,please try 'bcdedit /set hypervisorlaunchtype off' in prompt window with admin and reboot you PC, then try again.

This does not work

**Comment by MarioXs** [comment]
I also had the same problem
